### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SteveShengStar**

### CODE REVIEW

File: dump/admin/system.version.metadata.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"indexes":[{"v":{"$numberInt":"2"},"key":{"_id":{"$numberInt":"1"}},"name":"_id_"}],"uuid":"044a05f68f3a41929d0a5763ea329bd0","collectionName":"system.version"}
```
Comment (kevinbai0) [inline dump/admin/system.version.metadata.json:1]: should these dumps should be in gitignore?

File: backend/data/handlers/util.js [hunk 2]
```diff
@@ -57,7 +76,7 @@ util.handleWrapper = async (func) => {
 /**
  * Returns an object to be returned by an API endpoint with the success status and the data from a function call
  */
-util.resWrapper = async (func) => {
+util.resWrapper =  async (func) => {
```
Comment (kevinbai0) [inline backend/data/handlers/util.js:79]: small nit lol

File: pages/api/members/[id]/tasks.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+const data = require('../../../../backend/data/index');
+
+module.exports = async (req, res) => {
+    await data.initIfNotStarted();
+    
+    if (req.method === 'POST') {
+        const authStatus = await data.auth.checkAnyUser(req.headers['authorization'], res);
```
Comment (kevinbai0) [inline pages/api/members/[id]/tasks.js:7]: might be worth refactoring this out as middleware and wrapping as an higher order function (or I think Next.js has some support for middleware as well)

Comment (kevinbai0) [reply pages/api/members/[id]/tasks.js:7]: (the authentication part)

**Review by kevinbai0 — APPROVED** [review]
Overall, looks good - I'm assuming you've tested all this locally so far (and don't have much context with the new features you're building)

One thing to note is the update in `projects` before pushing this code (I might be wrong and everything could be fine). That part might break (but I don't think web or team hub really relies on Projects currently) so the easiest thing might be to wipe all projects from each `member` in prod so that you don't have any errors when working with the new projects schema.

**Review by Quinn1876 — COMMENTED** [review]
Didn't go through everything, but I took a look at the files you sent in the textfile plus a few others that I had looked at before I saw that file.

**Review by lynn-kim — APPROVED** [review]

**Pull request merged**